pytorch_lightning/trainer/connectors/accelerator_connector.py

```diff
@@ -791,15 +787,14 @@ def select_accelerator(self) -> Accelerator:
             return self.distributed_backend
 
         if self.use_gpu:
-            acc_cls = GPUAccelerator
+            accelerator = GPUAccelerator(benchmark=self.benchmark)
```

Comment:
This will break user who passed in a accelerator class. The line 407-411 handles when user pass in their own accelerator.  So this function is not guaranteed to be called

Reply:
Yes, the benchmark value won't be set in this case.